source/common/protobuf/utility.cc

```diff
@@ -373,7 +373,8 @@ class UnexpectedFieldProtoVisitor : public ProtobufMessage::ConstProtoVisitor {
                               Runtime::Loader* runtime)
       : validation_visitor_(validation_visitor), runtime_(runtime) {}
 
-  void onField(const Protobuf::Message& message, const Protobuf::FieldDescriptor& field) override {
+  void onField(const Protobuf::Message& message, const Protobuf::FieldDescriptor& field,
+               absl::Span<const Protobuf::Message* const>) override {
```

Comment:
Since this is not used right now maybe just skip passing parents in? We can always add this later.